suite/tests/api/ir_x86_3args_avx512_evex_mask.h

```diff
@@ -3312,25 +3312,23 @@ OPCODE(vsqrtpd_zhik7mem, vsqrtpd, vsqrtpd_mask, X64_ONLY, REGARG(ZMM16), REGARG(
 OPCODE(vsqrtpd_zhik7bcst, vsqrtpd, vsqrtpd_mask, X64_ONLY, REGARG(ZMM16), REGARG(K7),
        MEMARG(OPSZ_8))
 /* AVX512 BF16 */
-#if 0
-OPCODE(vcvtneps2bf16_xlok7xlo, vcvtneps2bf16, vcvtneps2bf16_mask, X64_ONLY, REGARG(XMM6),
-       REGARG(K7), REGARG(XMM0))
-OPCODE(vcvtneps2bf16_xhik7mem, vcvtneps2bf16, vcvtneps2bf16_mask, X64_ONLY, REGARG(XMM16),
-       REGARG(K7), MEMARG(OPSZ_16))
+OPCODE(vcvtneps2bf16_xlok7xlo, vcvtneps2bf16, vcvtneps2bf16_mask, X64_ONLY,
+       REGARG_PARTIAL(XMM6, OPSZ_8), REGARG(K7), REGARG(XMM0))
```

Comment:
Is there a reason for using this instruction on the low registers to be X64_ONLY?

Reply:
Updated.